packages/rocketchat-livechat/.app/client/views/messages.html

```diff
@@ -43,6 +43,9 @@
 				</div>
 				<div class="buttons">
 					<svg class="send-button" aria-label="{{_ "Send"}}" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1764 11q33 24 27 64l-256 1536q-5 29-32 45-14 8-31 8-11 0-24-5l-453-185-242 295q-18 23-49 23-13 0-22-4-19-7-30.5-23.5t-11.5-36.5v-349l864-1059-1069 925-395-162q-37-14-40-55-2-40 32-59l1664-960q15-9 32-9 20 0 36 11z"/></svg>
+					{{#if fileUploadEnabled}}
```

Comment:
please add a logic to hide the button if the textarea is not empty.

and what about changing the position of the button to above the textarea? like this:
![image](https://user-images.githubusercontent.com/8591547/40059999-e5813bae-582b-11e8-90c0-4e114a7493e6.png)